fastify.js

```diff
@@ -858,6 +863,14 @@ function fastify (options) {
       throw new FST_ERR_ERROR_HANDLER_NOT_FN()
     }
 
+    if (!options.allowErrorHandlerOverride && this[kErrorHandlerAlreadySet]) {
+      throw new FST_ERR_ERROR_HANDLER_ALREADY_SET()
+    } else if (this[kErrorHandlerAlreadySet]) {
+      FSTWRN004('It seems that you are overriding an errorHandler in the same scope, which can lead to subtle bugs.')
```

Comment:
This messagge is already included:

```suggestion
```

<img width="857" alt="image" src="https://github.com/user-attachments/assets/2fbe5ea8-60e6-4eb7-b43d-3735df396fc7" />